Mono.Cecil.Cil/PortablePdb.cs

```diff
@@ -249,6 +249,12 @@ public void Dispose ()
 				return;
 
 			WritePdbHeap ();
+
+			if (module_metadata.string_heap != pdb_metadata.string_heap) {
```

Comment:
Isn't this condition always true?
Also please move this to the local WriteTableHeap method.